edx-platform/pearson-pathways-theme/lms/templates/dashboard.html

```diff
@@ -156,6 +157,33 @@ <h1>${_("My Courses")}</h1>
             'SOCIAL_SHARING_SETTINGS',
             getattr(settings, 'SOCIAL_SHARING_SETTINGS', {})
           )
+
+          # MIT subscription information to be displayed on course cards.
+          mit_subscription = {}
```

Comment:
This variable should be defined before since it is passed to the course_listing template, otherwise, it will fire Reference before assignment. Does it make sense @Squirrel18?